### DISCUSSION TITLE
Browser extensions

### DISCUSSION THREAD

**Discussion by NiKola-UE** [seaweedfs/seaweedfs#5557] [Ideas]
What do you think about SeaweedFS extensions for different browsers (Chrome, Firefox, Opera, Edge, Yandex, Safari, etc.) that can also send and store different data and file tipes? 

Now, it may be exaggerating to compare IPFS and SeaweedFS protocols, but it kind of comes naturally; at least for us who know little about them and are just getting to know them. 

Anyway, the more implementations and applications, the better, right?

**Comment by chrislusf** [comment]
The idea is generic and not specific to SeaweedFS.

**Comment by NiKola-UE** [comment]
The idea may not be specific for now, but why couldn't it be? I'm not saying, maybe I explained it badly. But it's still open source software that doesn't have to be available only for operating systems, right? If browser extensions are not suitable, there is probably another way.

**Comment by NiKola-UE** [comment]
@chrislusf, 



Please close this discussion because I cannot it and there is really no point in keeping it open anymore. I understand that SeaweedFS is not used to store data, but I still think that extensions would be good when you need to cut out something that you don't want to download directly. Okay, that doesn't matter now. However, I will ask you to try to do something about 
[this issue ](https://github.com/seaweedfs/seaweedfs/issues/5603#issuecomment-2282272075) 
that I renamed, because I think that there should be parallel versions with full graphical and textual interfaces that would work with keyboard commands, which I explained there.
